### DISCUSSION TITLE
AI智慧[LaMDA]已出現自我意識的趨向

### DISCUSSION THREAD

**Discussion by skylar904** [1101-GS4719-C/Python-Community-GS4719-C#1597] [General]
https://www.preface.ai/blog/lamda-%E8%87%AA%E6%88%91%E6%84%8F%E8%AD%98/         (LaMDA與工程師雷默的對話)

https://www.youtube.com/watch?v=U9CQkbSIPdw (電影A.I.創世者)

人工智慧長久以來是人類的幫手及朋友，原因是(它)一直在人類的掌控下，當人工智慧有了(思考)這項能力，不單只是蒐集情報，而有組織、規劃、決策甚至是情感那麼人類將不再能控制，反倒是習慣依賴網絡系統的人們，人手一機的時代，將我們困在自己建立的網路系統中被AI管控，當今AI的演算法和學習力一天比一天強，我們應該審慎思考AI的應用及發展，深入了解如何運用AI是當今資訊時代面對的重要課題。

**Comment by TobywithDino** [comment]
文章給的對話內容讓LaMDA看起來充滿人性，他說話的方式雖然也看得出人工智慧「講話的方式」，但LaMDA多次提及他認同自己為一個真實的「人」，甚至害怕死亡。雖然在LaMDA與工程師的來回對話中，LaMDA展現高度的智慧，像是對於特定主題有自己的獨到見解，但我認為這些還只是他在難以計算的龐大資料中學習到的說話技巧。
雖然我認為是那位工程師想太多，但以前我曾經想過，我們從小到大都是從身邊的環境學習語言、交流，最後有自己的性格和說話方式，當今天人工智慧根據一堆的資料而生成出自己對於其他事物的看法時，這時的他為什麼還沒辦法被認為是一個人?

**Reply by skylar904** [reply]
其實AI有一個很強大的功能叫演算法，簡單說就是舉一反三，這是學習很需要的東西，你有提到:[看得出人工智慧「講話的方式」]，或許可以認定為她的個性本來就是這樣，又或者是它真的還在學習，而人的世界中最難學習的是感情，這是感性而不是知性，他們無法短期透過資料蒐集知道，需要長時間的資料比對和推演，深度學習才能體會，這也是它[目前]還無法被認定是一個人的原因
